pptx_blueprint/__init__.py

```diff
@@ -37,7 +37,27 @@ def replace_picture(self, label: str, filename: _Pathlike) -> None:
             label (str): label of the placeholder (without curly braces)
             filename (path-like): path to an image file
         """
-        pass
+        shapes_to_replace = self._find_shapes(label)
+        if not shapes_to_replace:
```

Comment:
This should be an Exception. Don't let errors pass silently!

Reply:
It is a good idea to handle the problem on a higher level. I will replace `return` by
```python
raise ValueError(f"The label '{label}' can't be found in the template.")
```